pkg/operator/sync_openshiftcontrollermanager_v311_00_test.go

```diff
@@ -280,179 +281,86 @@ func TestControllerDisabling(t *testing.T) {
 	}
 }
 
-func TestProgressingCondition(t *testing.T) {
-	ocmReplicas := int32(3)
-	rcmReplicas := int32(3)
-
-	happyPathOCMDeployment := func() *appsv1.Deployment {
-		return &appsv1.Deployment{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:       "controller-manager",
-				Namespace:  "openshift-controller-manager",
-				Generation: 100,
-			},
-			Spec: appsv1.DeploymentSpec{
-				Replicas: &ocmReplicas,
-			},
-			Status: appsv1.DeploymentStatus{
-				AvailableReplicas:  3,
-				ReadyReplicas:      3,
-				Replicas:           3,
-				UpdatedReplicas:    3,
-				ObservedGeneration: 100,
-			},
-		}
-	}
-
-	happyPathRCMDeployment := func() *appsv1.Deployment {
-		return &appsv1.Deployment{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:       "route-controller-manager",
-				Namespace:  "openshift-route-controller-manager",
-				Generation: 100,
-			},
-			Spec: appsv1.DeploymentSpec{
-				Replicas: &rcmReplicas,
-			},
-			Status: appsv1.DeploymentStatus{
-				AvailableReplicas:  3,
-				ReadyReplicas:      3,
-				Replicas:           3,
-				UpdatedReplicas:    3,
-				ObservedGeneration: 100,
-			},
-		}
-	}
+func TestAvailableCondition(t *testing.T) {
```

Comment:
Tests are generally not changed much. I just added testing Available and Degraded. The former tests are just duplicated for both operands. Also the reason is being checked now.